accesses/models/role.py

```diff
@@ -43,6 +43,13 @@ class Role(BaseModel):
     right_search_patients_by_ipp = models.BooleanField(default=False, null=False)
     right_search_patients_unlimited = models.BooleanField(default=False, null=False)
     right_search_opposed_patients = models.BooleanField(default=False, null=False)
+    # ORBIS
+    right_read_administrative_data = models.BooleanField(default=False, null=False)
```

Comment:
add a right `right_read_practitionner_data` (with default value False). 
We'll then use this access right to give practitioner and practitionerRole access in fhir.